test/wizeng/calls3.wasm.out

```diff
@@ -1,4 +1,4 @@
-func "main":
-  +8 ... +21 loop: 1005
-func "main":
+[0;36mfunc "main":
```

Comment:
For this, we'll probably want a "no colors" mode. I mentioned in person that there's a way to sense if the stdout is a terminal, but unfortunately that is platform-specific.